### PR TITLE
updated copyright years

### PR DESCRIPTION
**Pull request by CrazyWolf13**
[CrazyWolf13](https://github.com/CrazyWolf13) Medium [CrazyWolf13 /fix_copyright_year → Lissy93/dashy](https://github.com/Lissy93/dashy/tree/fix_copyright_year) Commits: 3 | Files Changed: 11 | Additions: 0 Label Unchecked Tasks [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=Lissy93&utm_campaign=badge_info)

**Category**: 
> Docs / Copyright year

**Overview**
> Updated all copyright years

**Issue Number** None

**Code Quality Checklist** _(Please complete)_
- [x] All changes are backwards compatible
- [x] All lint checks and tests are passing
- [x] There are no (new) build warnings or errors
- [ ] Bumps version, if new feature added

### CODE REVIEW

File: src/components/Configuration/AppInfoModal.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
       <!-- License -->
       <h3>{{ $t('app-info.license') }}</h3>
       {{ $t('app-info.license-under') }} <a href="https://github.com/Lissy93/dashy/blob/master/LICENSE">MIT X11</a>.
-      Copyright <a href="https://aliciasykes.com">Alicia Sykes</a> © 2021.<br>
+      Copyright <a href="https://aliciasykes.com">Alicia Sykes</a> © 2024.<br>
```
Comment (Lissy93) [inline src/components/Configuration/AppInfoModal.vue:33]: In the this file and `src/views/About.vue`, we might be able to make this dynamic with:
`{{new Date().getFullYear()}}`

**Review by Lissy93 — APPROVED** [review]
Thanks! :)

File: src/components/Configuration/AppInfoModal.vue (hunk 1 above)
Comment (CrazyWolf13) [reply src/components/Configuration/AppInfoModal.vue:33]: Sounds great! I'll try to implement that!

Comment (CrazyWolf13) [reply src/components/Configuration/AppInfoModal.vue]: @Lissy93 Done and commited!

**Review by Lissy93 — APPROVED** [review]

**Pull request merged**